lib/project_types/script/commands/create.rb

```diff
@@ -9,6 +9,8 @@ class Create < ShopifyCli::SubCommand
       end
 
       def call(args, _name)
+        return @ctx.puts(self.class.help) if args.include?('help')
```

Comment:
I don't think this is needed in any of the command class.  I think [here](https://github.com/Shopify/shopify-app-cli/blob/7e91e77f3b7ffe288b4a1b9b60773de79761392a/lib/shopify-cli/commands/help.rb#L59) already takes care of calling either `.help` or `.extended_help`.

Reply:
I found that removing this line caused the command (ie `shopify create script help` or `shopify deploy help`) to not display help anymore:
<img width="568" alt="Screen Shot 2020-06-09 at 7 49 18 AM" src="https://user-images.githubusercontent.com/64974039/84163040-e9b9c800-aa25-11ea-8cba-e065ea768beb.png">

Reply:
I think thats fine! Thats what happens in all the other commands.

I also don't personally think the CLI should have to support `shopify command help`. For example, `git push help` fails. But if we ever do want to support it, I'd imagine we'd make the change globally [here](https://github.com/Shopify/shopify-app-cli/blob/7e91e77f3b7ffe288b4a1b9b60773de79761392a/lib/shopify-cli/command.rb#L21) anyway